### DISCUSSION TITLE
SuiteQL syntax highlighting

### DISCUSSION THREAD

**Discussion by campnjim** [burkybang/SuiteAdvanced-Feedback#174] [Q&A] [answered]
Hi All,

I noticed that in the latest release video, the SuiteQL code has syntax highlighting. How do you turn that on? My code shows no highlighting.

Thanks

**Comment by burkybang** [comment, marked as answer]
I just finished replying to you on YouTube 🙂

Code syntax highlighting themes are not supported in code editor fields. They only work in execution logs and read only text fields if they contain JSON or XML.

**Comment by campnjim** [comment]
Hi Adam,
That's too bad, but thanks for the quick response to my question.




Jim Roberts
IT/MIS Manager
***@***.***
Toll Free:800-446-2500
Phone (Intl +1):
630-529-7111
www.qmiusa.com
Leaders in Forced Entry ​and Ballistic Security
This email may contain material that is confidential, privileged and/or attorney work product for the sole use of the intended recipient. Any review, reliance or distribution by others or forwarding without express permission is strictly prohibited. If you are not the intended recipient, please contact the sender and delete all copies
________________________________
